script/1_0_2current_en_script/Arcueid/Day 1/01_00_ARC01_3_2.txt

```diff
@@ -32,5 +32,5 @@ Then, I was involved in an accident that left me with a physical defect, and Mak
 He said, 'Even if you're the eldest son, you can't be the successor as a weak man who might die at any time.'
 I defied his expectations and managed to recover, but by that time, my younger sister Akiha was chosen as the heir.
 <Page10>
-Up until then, Akiha had been raised as a 'Tohno daughter', but once I had dropped out, she was raised even more strictly.
-I played with Akiha in the mansion garden until I turned ten years old, and got involved in the accident. I haven't met her since then.
+Akiha was already being raised as 'A daughter of the Tohno family', but once I had dropped out, she was raised even more strictly.//それまで『遠野の娘』として育てられていた秋葉は、俺が脱落した事により、いっそう厳しく育てられる事になった。
```

Comment:
but once I dropped out, it became even more strict.

Reply:
Hrm, I feel like the subject for `it` is tenuous in that alternate for whatever reason. 
> Akiha was already being raised as a '...', but once I had dropped out, her upbringing became even stricter. // ?